src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -3445,6 +3445,82 @@ GUI_TEST_CLASS_DEFINITION(test_7697) {
+    //     Expected: the ugenedb file is successfully created in the default directory and opens without problems, the
+    //         Assembly Browser shows position 45 890 375 with coverage 2316 as the first well-covered region.
+    QString sandboxPath = QFileInfo(sandBoxDir).canonicalFilePath();
+    int requiredNumOfChars = 250 - sandboxPath.size();
+    QString longPath = sandboxPath + QString::fromWCharArray(L"/\u221E").repeated(requiredNumOfChars / 2);
+    CHECK_SET_ERR(QDir().mkpath(longPath), "Failed to create dir '" + longPath + "'");
+
+    class WorkflowOutputScenario : public CustomScenario {
+        QString path;
+
+    public:
+        explicit WorkflowOutputScenario(const QString& path)
+            : path(path) {
+        }
+        void run(GUITestOpStatus& os) override {
+            QWidget* dialog = GTWidget::getActiveModalWidget(os);
+            GTLineEdit::setText(os, "pathEdit", path, dialog, false, true);
+            GTUtilsDialog::clickButtonBox(os, dialog, QDialogButtonBox::Ok);
+        }
+    };
+    GTUtilsDialog::waitForDialog(os, new Filler(os, "StartupDialog", new WorkflowOutputScenario(longPath)));
+    GTFileDialog::openFile(os, testDir + "_common_data/scenarios/_regression/7700/bwa.uwl");
+    GTUtilsWorkflowDesigner::checkWorkflowDesignerWindowIsActive(os);
+
+    GTUtilsWorkflowDesigner::click(os, "Read File URL(s)");
+    GTUtilsWorkflowDesigner::setDatasetInputFiles(
+        os, {testDir + "_common_data/bwa/control-chr21.fastq", testDir + "_common_data/bwa/nrsf-chr21.fastq"});
+
+    GTUtilsWorkflowDesigner::click(os, "Map Reads with BWA");
+    GTUtilsWorkflowDesigner::setParameter(os,
+                                          "Reference genome",
+                                          testDir + "_common_data/bwa/NC_000021.gbk.fa",
+                                          GTUtilsWorkflowDesigner::valueType::lineEditWithFileSelector);
+
+    GTUtilsWorkflowDesigner::runWorkflow(os);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    GTUtilsDialog::waitForDialog(os, new ImportBAMFileFiller(os));
+    GTUtilsDashboard::clickOutputFile(os, "out.sam");
+    GTUtilsAssemblyBrowser::checkAssemblyBrowserWindowIsActive(os);
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    QString coveredRegionsText;
+    class GetLabelTextScenario : public CustomScenario {
+        QLabel* label;
+        QString& text;
+
+    public:
+        GetLabelTextScenario(QLabel* label, QString& text)
+            : label(label), text(text) {
+        }
+        void run(GUITestOpStatus&) override {
+            text = label->text();
+        }
+    };
+    GTThread::runInMainThread(
+        os, new GetLabelTextScenario(GTWidget::findLabel(os, "CoveredRegionsLabel"), coveredRegionsText));
+    CHECK_SET_ERR(coveredRegionsText.contains("<td><a href=\"0\">45 890 375</a></td><td align=\"center\">2 316</td>"),
+                  "Position 45 890 375 with 2 316 coverage isn't the first well-covered region");
```

Comment:
Print the current coveredRegionsText value in the error message

Reply:
✓